### PR TITLE
Use RECORD env var to control record mode

### PR DESCRIPTION
**Pull request by NouemanKHAL**
- `RECORD=true` for recording, `RECORD=false` for replaying and `RECORD=none` for using the real API without recording
- Default values for environment vars will be taken care of in #69

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -41,9 +41,18 @@ def before_record_response(response):
     return before_record_response
 
 
+def get_record_mode():
+    record = os.getenv("RECORD", "none")
+    if record.lower() == "true":
+        return "all"
+    else:
+        return "none"
```
Comment (zippolyte) [inline tests/conftest.py:49]: There are three cases `RECORD=true` for recording, `RECORD=false` for replaying and `RECORD=none` for using the real API without recording

**Review by zippolyte — CHANGES_REQUESTED** [review]

File: tests/conftest.py (hunk 1 above)
Comment (NouemanKHAL) [reply tests/conftest.py:49]: I don't think there's an option to not record : https://vcrpy.readthedocs.io/en/latest/usage.html#record-modes
Unless you pass the --disable-recording flag

Comment (zippolyte) [reply tests/conftest.py:49]: We have these 3 modes in the API clients, so let's have them here too

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -35,10 +35,10 @@ jobs:
             - name: Install datadog-sync-cli
               run: pip install .\[tests\]
             - name: Run unit tests
-              run: pytest --record-mode=none -ra -v
+              run: pytest -ra -v
             - name: Run integration tests
               if: contains(github.event.pull_request.labels.*.name, 'ci/integration')
-              run: pytest --disable-recording -ra -v -m "integration"
+              run: RECORD=true pytest -ra -v -m "integration"
```
Comment (jirikuncar) [inline .github/workflows/test.yml:41]:
```suggestion
              run: RECORD=none pytest -ra -v -m "integration"
```

**Review by zippolyte — APPROVED** [review]

**Pull request merged**